adapters/Survey/index.test.js

```diff
@@ -0,0 +1,42 @@
+import baseAdapter from 'adapters/Base';
+import surveyAdapter from '.';
+
+describe('surveyAdapter', () => {
+  describe('fetchSurveys', () => {
+    describe('given the current page and an authorization header', () => {
```

Comment:
```suggestion
    describe('given the current page is 1 and an authorization header', () => {
```

or 

```suggestion
    describe('given a current page param and an authorization header', () => {
```

Reply:
done in e9f3815051be75cb8f3734494b6d05c9fc389acf